src/main/java/carpet/script/api/Auxiliary.java

```diff
@@ -138,10 +138,7 @@ public static void apply(Expression expression)
             String rawString = lv.get(0).getString();
             ResourceLocation soundName = InputValidator.identifierOf(rawString);
             Vector3Argument locator = Vector3Argument.findIn(lv, 1);
-            if (cc.registry(Registries.SOUND_EVENT).get(soundName) == null)
-            {
-                throw new ThrowStatement(rawString, Throwables.UNKNOWN_SOUND);
-            }
+
             Holder<SoundEvent> soundHolder = Holder.direct(SoundEvent.createVariableRangeEvent(soundName));
```

Comment:
Wondering what does this return now, given holder stuff sounds like registry stuff and now the sound may not be in the registry. Does it create some temporary one? (mostly concerned about the intrusive holder stuff on freeze)